department-of-veterans-affairs/abd-vro
# MCP-2672 MCP-2699 Font Config + Missing Webfont Fix

## What was the problem?

When I updated all the styling/font imports to be added to the repo rather than downloaded on demand, it caused an issue with FontAwesome. I only imported the `.css` file but that file was dependent on a `webfonts` folder as well.
Secondly, we were getting `Fontconfig error: No writable cache directories` errors that were due to the `tron` user not having access to any font cache location.

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]


- Add the `webfonts` to the repo as well
- Change docker to create a home directory for `tron` which is where the font cache will go

## How to test this PR
- Run E2E or generate a PDF. Shouldn't see any failed PDFs or errors regarding font config


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
